composer.lock

```diff
@@ -4,32 +4,33 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "0e4428b976a1e1835abdff64014ad723",
+    "content-hash": "9147cdc691d11f68d6fbae8012f036bd",
     "packages": [
         {
             "name": "doctrine/cache",
-            "version": "v1.7.1",
+            "version": "1.10.2",
```

Comment:
It looks like you updated all locked dependencies. Would it make sense to just update psalm with dependencies?

Reply:
Should be k now